### PR TITLE
IMP kafka reader and writer + unittests

### PR DESCRIPTION
**Pull request by skliarpawlo**
# WHY
This functionality as used in our infra for a while.

# WHAT
Kafka writer and reader

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,72 +1,11 @@
...
\ No newline at end of file
+## 1.1.0
+* Kafka reader and writer.
+* Kafka fixtures.
+
+## 1.0.0
+* Initial open-source release.
+* Features:
+ - Convenient dependencies setup: packages, jars, UDFs.
+ - Reader and Writer unique interface: parquet, csv, mysql, cassandra, elastic.
```
Comment (drudim) [inline CHANGELOG.md]: unique?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,72 +1,11 @@
...
-* Initial version: basic convenient functions and Sparkly context
-* Readers for elastic, cassandra, csv
\ No newline at end of file
+## 1.1.0
+* Kafka reader and writer.
+* Kafka fixtures.
+
+## 1.0.0
+* Initial open-source release.
+* Features:
```
Comment (drudim) [inline CHANGELOG.md]: From my FB post:
> - Declarative definition of application dependencies (spark packages, jars, UDFs)
> - Readers and writers for ElasticSearch, Cassandra, MySQL
> - DSL for interaction with Apache Hive Metastore

Comment (skliarpawlo) [inline CHANGELOG.md]: Fixed

**Comment by skliarpawlo** [comment]
@drudim ready for review

**Pull request merged**